### DISCUSSION TITLE
cannot find the module 'react'

### DISCUSSION THREAD

**Discussion by Angryman18** [vercel/next.js#45567] [Help]
what is the problem whenever i import something from 'react-redux' i always get to see this error. i am using redux toolkit with redux-persist. please help me

![Screenshot from 2023-02-04 10-10-02](https://user-images.githubusercontent.com/63530626/216748847-01fba340-fc39-4e4c-82cb-7ab165e5e0c0.png)

**Comment by ciruz** [comment]
Is the correct react version installed?

Maybe remove `/node_modules` and install all dependencies again.

**Reply by Angryman18** [reply]
Hi this has solved. Thankq
